TextAnalytics/pitchfork_data/pitchfork_language_model.ipynb

```diff
@@ -0,0 +1,510 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial we are going to implement a transfer learning model for text version of the ULMfit. \n",
+    "We are first running the language model needed for the classifier."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "# username \n",
+    "os.environ['KAGGLE_USERNAME'] = \"jlealtru\" \n",
+    "# key\n",
+    "os.environ['KAGGLE_KEY'] = \"6c3a4d6b4d8e7804780d6cb02879ac53\""
```

Comment:
@jlealtru never post secrets/keys in source code. You have a couple options. The easiest, although not safest, is creating a separate file and import that file in and reference the variable name only in the code. Alternatively, you can leverage something like Azure Key Vault (easy to use, super powerful - think of OnePassword or LastPass except at scale/programatically)